app/components/contact/contact-item/contact-item-modal.js

```diff
@@ -0,0 +1,50 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { isBlank } from '@ember/utils';
+
+export default class ContactContactItemModalComponent extends Component {
+  @service currentSession;
+  @service store;
+
+  @tracked contact;
+
+  constructor() {
+    super(...arguments);
+    if (this.args.contact) {
+      this.contact = this.args.contact;
+    } else {
+      const creator = this.currentSession.organization;
+      const telephone = this.store.createRecord('telephone', {creator});
+      const mailAddress = this.store.createRecord('mail-address', {creator});
+      this.contact = this.store.createRecord('contact-item', {telephone, mailAddress});
+    }
+  }
+
+  get isAddDisabled() {
+    return !this.contact.givenName &&
+           !this.contact.familyName &&
+           !this.contact.telephone &&
+           !this.contact.mailAddress &&
+           !this.contact.organizationName;
```

Comment:
De [`isPresent` of `isBlank`](https://api.emberjs.com/ember/release/functions/@ember%2Futils/isPresent) utils zijn hier geschikter om values met enkel whitespaces niet als `true` te aanzien.